api/src/main/proto/input/reqreply/message/create.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package rys.ajaxpetproject.request.message.create.proto;
+
+import "commonmodels/message/message.proto";
+import "commonmodels/error/error.proto";
+
+option java_multiple_files = true;
+
+message MessageCreateRequest {
+  commonmodels.message.proto.Message message = 1;
+  string chatId = 2;
+}
+
+message MessageCreateResponse {
+
+  oneof response {
+    Success success = 1;
+    Failure failure = 2;
+  }
+
+  message Success {
+    string result = 1;
+  }
+
+  message Failure {
+    optional string message = 1;
+    commonmodels.error.proto.Error error = 2;
```

Comment:
the goal is to have here a list of business errors, general 'error' doesn't say anything about what happened

Reply:
Yes, but I will took time to write many errors and test them. So in order to not doing copy-paste actions I've left just error